test/src/api/commands/cookies/testCookiesErrors.js

```diff
@@ -33,7 +33,10 @@ describe('getCookiesErrors', function() {
         report_command_errors: true,
         backwards_compatibility_mode: true,
         webdriver: {
-          start_process: false
+          start_process: false,
+          timeout_options: {
+            retry_attempts: 0
+          }
```

Comment:
If we are having to specifically mention the `timeout_options` property everywhere, won't it affect our users as well? Because I feel most of our users won't be using this property already in their configuration file unless they have something to do with retries.

Reply:
For these tests, the `timeout_options` will not be necessary if I setup the mock servers properly. It is easier to test with `timeout_options`. The behaviour for the user does not change.